staging/src/k8s.io/apiserver/pkg/apis/config/register.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var (
+	// SchemeBuilder points to a list of functions added to Scheme.
+	SchemeBuilder = runtime.NewSchemeBuilder(addKnownTypes)
+	// AddToScheme adds this group to a scheme.
+	AddToScheme = SchemeBuilder.AddToScheme
+)
+
+// GroupName is the group name use in this package.
+const GroupName = "apiserver.config.k8s.io"
```

Comment:
This is a reasonable group name, but is there any discussion of this in a KEP or proposal that I can ready?  Or just adhoc PR discussion?

Reply:
I'm afraid that most of this PR is based on adhoc discussions, where it was discovered that a new API group will be necessary.
I believe https://github.com/kubernetes/kubernetes/issues/61599#issuecomment-375711161 is the first mention of that API group name.

Reply:
@smarterclayton is there something I can do right now to improve the situation so that the way this is going is clearer?

Reply:
`<component>.config.k8s.io` was the pattern settled on in https://github.com/kubernetes/community/blob/master/keps/sig-cluster-lifecycle/0014-20180707-componentconfig-api-types-to-staging.md#timeframe-and-implementation-order

the `apiserver.config.k8s.io` group was used for a parallel use case of an [admission-related config file](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/config/apis/webhookadmission/v1alpha1/register.go#L24-L25) that any generic apiserver could consume